packages/lit-html/src/lit-html.ts

```diff
@@ -494,17 +494,24 @@ const getTemplateHtml = (
     //  4. We're somewhere else inside the tag. If we're in attribute name
     //     position (attrNameEndIndex === -2), add a sequential suffix to
     //     generate a unique attribute name.
+
+    // Detect a binding next to self-closing tag end and insert a space to
+    // separate the marker from the tag end:
+    const end =
+      regex === tagEndRegex && strings[i + 1].startsWith('/>') ? ' ' : '';
```

Comment:
Nit: ? Could we simplify this and just always insert a space?

Reply:
We can't because the current regex is `tagEndRegex` for unquoted attributes, _and_ we also support multiple bindings in an unquoted attribute now, like `<div foo=${a}${b}></div>`. We can't insert a space after the first binding as that'll close the attribute early.